experiments/commands.py

```diff
@@ -0,0 +1,234 @@
+r"""Generate commands to run training scripts with different configs.
+
+This program takes as input a set of config files and outputs a command
+for each config file. Each command runs a training script using the
+associated config file. By default, each command uses the same random seed
+for training. However, if multiple random seeds are specified, the program
+generates multiple commands for each config file (one for each random seed).
+The commands can be executed by copying and pasting them into a command line
+or by piping them to another command line utility. This program is helpful
+for running a large number of training scripts with different config files
+using multiple random seeds.
+
+For example, we can run:
+
+python commands.py \
+    --name=run0 \
+    --cfg_pattern=../benchmarking/*ai*_seals_walker_*.json \
+    --output_dir=output
+
+And get the following commands printed out:
+
+python -m imitation.scripts.train_adversarial airl \
+    --capture=sys --name=run0 \
+    --file_storage=output/sacred/$USER-cmd-run0-airl-0-a3531726 \
+    with ../benchmarking/example_airl_seals_walker_best_hp_eval.json \
+    seed=0 logging.log_root=output
+
+python -m imitation.scripts.train_adversarial gail \
+    --capture=sys --name=run0 \
+    --file_storage=output/sacred/$USER-cmd-run0-gail-0-a1ec171b \
+    with ../benchmarking/example_gail_seals_walker_best_hp_eval.json \
+    seed=0 logging.log_root=output
+
+We can execute commands in parallel by piping them to GNU parallel:
+
+python commands.py ... | parallel -j 8
+
+If the --remote flag is enabled, then the program prints out commands
+to run training scripts in containers on the Hofvarpnir cluster.
+
+For example, we can run:
+
+python commands.py \
+    --name=run0 \
+    --cfg_pattern=../benchmarking/example_bc_seals_half_cheetah_best_hp_eval.json \
+    --output_dir=/data/output \
+    --remote
+
+And get the following command printed out:
+
+ctl job run --name $USER-cmd-run0-bc-0-72cb1df3 \
```

Comment:
This defaults to requesting a GPU. Do you want that? Can disable by `--gpu 0` I believe